templates/services/merge/distributedValidator/charon.tmpl

```diff
@@ -0,0 +1,24 @@
+{{/* charon.tmpl */}}
+{{ define "distributedValidator" }}
+  charon:
+    image: obolnetwork/charon:${DV_IMAGE_VERSION}
+    environment:
+      - CHARON_BEACON_NODE_ENDPOINTS=${CC_API_URL}
+      - CHARON_LOG_LEVEL=${DV_LOG_LEVEL:-info}
+      - CHARON_LOG_FORMAT=${DV_LOG_FORMAT:-console}
+      - CHARON_P2P_RELAYS=${DV_P2P_RELAYS:-https://0.relay.obol.tech}
+      - CHARON_P2P_EXTERNAL_HOSTNAME=${DV_INSTANCE_NAME:-charon}
+      - CHARON_P2P_TCP_ADDRESS=0.0.0.0:{{.DVDiscoveryPort}}
+      - CHARON_VALIDATOR_API_ADDRESS=0.0.0.0:{{.DVApiPort}}
```

Comment:
We could consider [this PR](https://github.com/ObolNetwork/lido-charon-distributed-validator-node/pull/46) and see if a similar level of safety could be followed for this line and the CHARON_MONITORING_ADDRESS. It depends on the downstream validator using the container name to find charon instead of an IP and port though. This should be okay i assume. 
```suggestion
      - CHARON_VALIDATOR_API_ADDRESS=127.0.0.1:{{.DVApiPort}}
```